### DISCUSSION TITLE
GitHub Roadmap : Self-hosted runner auto-scaling support for Kubernetes

### DISCUSSION THREAD

**Discussion by Moser-ss** [actions/actions-runner-controller#1734] [General]
Hey,

I found in the GitHub Public Roadmap a ticket to work on the [Self-hosted runner auto-scaling support for Kubernetes](https://github.com/github/roadmap/issues/555). I am thinking about the impact of that ticket on this project.

Will we have two solutions to support auto-scaling of self-hosted runners on Kubernetes

**Comment by nebuk89** [comment]
👋 hello :) I am the PM at GitHub looking after the roadmap item you flagged.

We are working with the maintainers of ARC to ensure there will only be one solution as we go forward 🙌 watch this space for updates coming in the next few weeks 🎉

**Reply by iniinikoski** [reply]
> 👋 hello :) I am the PM at GitHub looking after the roadmap item you flagged.
> 
> We are working with the maintainers of ARC to ensure there will only be one solution as we go forward 🙌 watch this space for updates coming in the next few weeks 🎉

Wow! Thanks @Moser-ss for bringing this here (I was just about to you were way faster :))

Looking forward for this @nebuk89! Awesome!

**Comment by Diontsoumas** [comment]
Very exciting!
